sdk/js/src/utils/base64.ts

```diff
@@ -1,3 +1,6 @@
+import fromBase64 from 'es-arraybuffer-base64/Uint8Array.fromBase64';
```

Comment:
Wouldn't importing this package already make it possible to use `Uint8Array.fromBase64` directly? when the polyfill is not needed anymore we just remove one line.